core/token-pooling/redis-token-persistence.integration.js

```diff
@@ -18,17 +17,15 @@ describe('Redis token persistence', function() {
 
   const key = 'tokenPersistenceIntegrationTest'
 
-  let client
+  let redis
   beforeEach(async function() {
-    client = redis.createClient()
-    const del = promisify(client.del).bind(client)
-    await del(key)
+    redis = new Redis()
+    await redis.del(key)
```

Comment:
nice to not have to promisify everything here 👍

Reply:
For sure!